addons/sale/tests/test_sale_order.py

```diff
@@ -13,6 +15,9 @@ class TestSaleOrder(TestCommonSaleNoChart):
     def setUpClass(cls):
         super(TestSaleOrder, cls).setUpClass()
 
+        cls.Pricelist = cls.env['product.pricelist']
+        cls.PricelistItem = cls.env['product.pricelist.item']
+        cls.SaleOrderLine = cls.env['sale.order.line']
```

Comment:
Don't put the Model on the class. Rather use ` Pricelist = self.env[...]`. If the model inherits 'mail.thread', give it the `tracking_disable` context key.